src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraGlobalUIConfig.java

```diff
@@ -35,22 +35,16 @@
 @Component
 public class JiraGlobalUIConfig extends UIConfig {
     public static final String LABEL_URL = "Url";
-    public static final String LABEL_USER_NAME = "Username";
-    public static final String LABEL_ACCESS_TOKEN = "Access Token";
+    public static final String LABEL_USER_NAME = "Admin Username";
+    public static final String LABEL_ACCESS_TOKEN = "Admin Api Token";
     public static final String LABEL_CONFIGURE_PLUGIN = "Configure Jira Cloud plugin";
-    public static final String LABEL_ADMIN_USERNAME = "Admin Username";
-    public static final String LABEL_ADMIN_ACCESS_TOKEN = "Admin Access Token";
 
     public static final String DESCRIPTION_URL = "The URL of the Jira Cloud server.";
-    public static final String DESCRIPTION_USER_NAME = "The email address used to log into the Jira Cloud server that has generated the access token.";
-    public static final String DESCRIPTION_ACCESS_TOKEN = "The access token used to send API requests to the Jira Cloud server.";
+    public static final String DESCRIPTION_USER_NAME = "The email address of the admin used to log into the Jira Cloud server that has generated the api token.";
+    public static final String DESCRIPTION_ACCESS_TOKEN = "The admin api token used to send API requests to the Jira Cloud server.";
```

Comment:
Can we also update the variable name?

Reply:
This string has both "api" and "API" in it.

Reply:
Any references to API should be in all caps since it is an acronym.